replays.py

```diff
@@ -22,6 +24,16 @@
 print('Loaded replay app')
 
 
+@bp.route('/', methods=['GET', 'POST'])
+def replays_home():
+    engine, Session = startup()
```

Comment:
Don't use `startup()`, use `current_app.config['db']` as `Session` in this case. Prevents unnecessary connections to the DB.